pkg/ccl/backupccl/restore_online.go

```diff
@@ -430,7 +430,7 @@ func (r *restoreResumer) maybeCalculateTotalDownloadSpans(
 
 	if err := r.job.NoTxn().Update(ctx, func(txn isql.Txn, md jobs.JobMetadata, ju *jobs.JobUpdater) error {
 		md.Progress.GetRestore().TotalDownloadRequired = total
-		md.Progress.RunningStatus = fmt.Sprintf("Downloading %s of restored data...", sz(total))
+		md.Progress.RunningStatus = fmt.Sprintf("Downloading %s of backup data...", sz(total))
```

Comment:
well, the data has already been restored via the linking phase! i don't think we should change this?

Reply:
Ah that's what the rationale for it was. @alicia-l2 what are your thoughts on this?

Reply:
Oh this makes sense oops! Thanks for clarifying

Reply:
Reverted this back to the original wording